hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationServiceImpl.java

```diff
@@ -444,6 +444,15 @@ public void onMemberLeft(MemberImpl member) {
         invocationMonitor.onMemberLeft(member);
     }
 
+    /**
+     * Cleans up heartbeats and fails invocations for the given endpoint.
+     *
+     * @param endpoint the endpoint that has left
+     */
+    public void onEndpointLeft(Address endpoint) {
```

Comment:
Could be declared on `InternalOperationService` to avoid downcasting

Reply:
Got it!